src/Model.jl

```diff
@@ -1,5 +1,21 @@
 # Contains the physical specification of the model
 
+using Unitful
+import Unitful.uconvert
+using UnitfulAtomic
+
+# Convert a temperature to an energy via the Boltzmann constant.
+function uconvert(a::Unitful.EnergyUnits, x::Unitful.Quantity{T,Unitful.𝚯,U}) where {T <: Real, U <: Unitful.Units}
```

Comment:
This is a bit over-typed (in Julia you try to be as loose on specifying the types as possible). You can simply do `where {T, U}` and that's it.